Description: Switch to the sha1 crate
Author: Mike Hommey <mh@glandium.org>
Origin: upstream
Applied-Upstream: 1df6829ca4b8b40bb02cf223c0bd83ac82ecc92e
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: git-cinnabar/Cargo.toml
===================================================================
--- git-cinnabar.orig/Cargo.toml
+++ git-cinnabar/Cargo.toml
@@ -160,7 +160,7 @@ default-features = false
 [dependencies.semver]
 version = "1.0"
 
-[dependencies.sha-1]
+[dependencies.sha1]
 version = "0.10"
 
 [dependencies.shared_child]
